src/messages/components/Attachments.vue

```diff
@@ -0,0 +1,465 @@
+            flat
+            round
+            size="sm"
+            color="red"
+            class="bg-white z-top absolute-top-right attachment--action q-ma-xs"
+            :title="$t('BUTTON.DELETE')"
+            @click="removeAttachment(attachment)"
+          />
+
+          <QBtn
+            v-if="!props.edit && attachment.urls.download"
+            icon="fas fa-download"
+            flat
+            round
+            size="sm"
+            color="white"
+            text-color="primary"
+            class="bg-white z-top absolute-bottom-right attachment--action q-ma-xs"
+            :href="attachment.urls.download"
+            :title="$t('BUTTON.DOWNLOAD')"
+          />
+
+          <template
+            v-if="attachment._img"
+          >
+            <img
+              v-if="props.edit"
+              :src="attachment._img.src"
+              class="rounded-borders image"
+            >
+            <a
+              v-else
+              :href="attachment.urls.original"
+            >
+              <img
+                :src="attachment._img.src"
+                class="rounded-borders image cursor-pointer"
+                @click.stop.prevent="openGallery(attachment.id)"
+              >
+            </a>
+          </template>
+          <template v-else>
+            <QItem
+              class="q-pa-sm"
+            >
+              <QItemSection
+                v-if="attachment._contentTypeInfo.icon"
+                avatar
+                class="q-pr-sm"
+                style="min-width: auto;"
+              >
+                <QIcon
+                  :name="attachment._contentTypeInfo.icon"
+                  size="md"
+                  color="grey"
+                />
+              </QItemSection>
+              <QItemSection>
+                <div class="text-subtitle1 ellipsis">
+                  {{ attachment.filename }}
+                </div>
+                <div class="text-caption ellipsis">
+                  <span v-if="attachment._contentTypeInfo.name">
+                    {{ attachment._contentTypeInfo.name }}
+                  </span>
+                  {{ attachment._sizeLabel }}
+                </div>
+              </QItemSection>
+            </QItem>
+          </template>
+        </QCard>
+      </div>
+    </QItemSection>
+    <AttachmentGallery
+      v-if="galleryOpenId"
+      :attachments="attachments"
+      :selected-attachment-id="galleryOpenId"
+      @close="galleryOpenId = null"
+    />
+  </QItem>
+</template>
+
+<script setup>
+import {
+  QItem,
+  QItemSection,
+  QBtn,
+  QCard,
+  QIcon,
+  format,
+} from 'quasar'
+import { onUnmounted, ref, computed } from 'vue'
+
+import { showToast } from '@/utils/toasts'
+import { isViewableImageContentType } from '@/utils/utils'
+
+import AttachmentGallery from '@/messages/components/AttachmentGallery.vue'
+
+const { humanStorageSize } = format
+
+const MAX_ATTACHMENT_COUNT = 6
+const MAX_ATTACHMENT_SIZE = 10 * 1024 * 1024
```

Comment:
out of sync with backend, maybe could be part of bootstrap/config endpoint?

Reply:
done, added to backend :)